pkg/operator/sync.go

```diff
@@ -633,6 +639,60 @@ func (optr *Operator) syncCustomResourceDefinitions() error {
 				return err
 			}
 		}
+		if !equality.Semantic.DeepEqual(c.Spec, current.Spec) || !equality.Semantic.DeepEqual(c.Status, current.Status) {
```

Comment:
Interesting, I'm surprised we got this long without having to do this... how does other operators handle having to update the CRD/CR? Would it normally happen in the same loop?